lib/chef/platform/service_helpers.rb

```diff
@@ -126,7 +131,7 @@ def extract_systemd_services(command)
           # this splits off the suffix after the last dot to return "sshd"
           services += services.select {|s| s.match(/\.service$/) }.map { |s| s.sub(/(.*)\.service$/, '\1') }
         rescue Mixlib::ShellOut::ShellCommandFailed
```

Comment:
Are there other exit statuses besides `1` that we should maybe not be rescuing (or treating as success, anyway)? Perhaps adding `return: [0,1]` to the `shell_out` call would be safer?